src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -58,16 +63,67 @@ def _format_group(item):
     """Returns an ordered dictionary of the container registry.
     :param dict item: The container registry object
     """
-    registry_info = {_registry_map[key]: str(item[key]) for key in item if key in _registry_map}
-
-    if 'sku' in item and 'name' in item['sku']:
-        registry_info['SKU'] = item['sku']['name']
-    if 'username' in item:
-        registry_info['USERNAME'] = item['username']
-    if 'passwords' in item:
-        if item['passwords'] and 'value' in item['passwords'][0]:
-            registry_info['PASSWORD'] = item['passwords'][0]['value']
-        if len(item['passwords']) > 1 and 'value' in item['passwords'][1]:
-            registry_info['PASSWORD2'] = item['passwords'][1]['value']
-
-    return OrderedDict(sorted(registry_info.items(), key=lambda t: _order_map[t[0]]))
+    table_info = {_property_map[key]: str(item[key]) for key in item if key in _property_map}
+
+    try:
+        table_info['SKU'] = item['sku']['name']
+    except (KeyError, TypeError):
+        pass
+
+    try:
+        table_info['USERNAME'] = item['username']
+    except (KeyError, TypeError):
+        pass
+
+    try:
+        table_info['PASSWORD'] = item['passwords'][0]['value']
+    except (KeyError, TypeError, IndexError):
+        pass
+
+    try:
+        table_info['PASSWORD2'] = item['passwords'][1]['value']
+    except (KeyError, TypeError, IndexError):
+        pass
+
+    try:
+        # Only show ID if it is not an ARM resource ID
+        table_info['ID'] = item['id'] if '/subscriptions/' not in item['id'] else None
```

Comment:
`subscriptions` case sensitive?

Reply:
It is supposed to be `subscriptions` but making it case-insensitive is fine too.